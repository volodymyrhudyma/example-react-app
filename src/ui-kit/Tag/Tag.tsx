import React, { ReactNode } from 'react';
import { Wrapper } from './styled/Tag.styled';

interface ITagProps {
  children: ReactNode;
}

const Tag = (props: ITagProps) => <Wrapper {...props} />;

export default Tag;
