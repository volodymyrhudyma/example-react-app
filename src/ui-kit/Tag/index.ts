import Component from './Tag';

export default Component;
