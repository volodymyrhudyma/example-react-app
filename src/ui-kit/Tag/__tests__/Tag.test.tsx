import React from 'react';
import { shallow } from 'enzyme';
import Tag from '../Tag';

describe('Tag component', () => {
  it('should render correctly', () => {
    const component = shallow(<Tag>Tag</Tag>);
    expect(component).toMatchSnapshot();
  });
});
