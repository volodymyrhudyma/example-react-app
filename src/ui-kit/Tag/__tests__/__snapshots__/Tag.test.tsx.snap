// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag component should render correctly 1`] = `
<styled.div>
  Tag
</styled.div>
`;
